core/src/main/java/io/questdb/cutlass/parquet/CopyExportRequestTask.java

```diff
@@ -204,4 +272,202 @@ public String getName() {
             return name;
         }
     }
+
+    @FunctionalInterface
+    public interface StreamWriteParquetCallBack {
+        void onWrite(long dataPtr, long dataLen) throws Exception;
+    }
+
+    public class StreamPartitionParquetExporter implements Mutable {
```

Comment:
this is also very confusing class declaration - this class should be closable, not mutable. It is not re-usable after `clear()` call, which releases lists

Reply:
`StreamPartitionParquetExporter` is reused because it's a member of `CopyExportRequestTask`, and that task is reused, so the exporter was being reused too.